### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by EckyYakov**
Added in all relevant scripts and licenses for the initial commit.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,201 +1,21 @@
-                                 Apache License
```
Comment (v1k0d3n) [inline LICENSE]: apache license please.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,82 @@
-# brigade-robot-framework
-This repository uses Azure/Brigade for running in cluster (Kubernetes) Robot Framework tests.
+# Robot-Brigade
```
Comment (v1k0d3n) [inline README.md]: brigade-robot-framework

**Review by v1k0d3n — COMMENTED** [review]
tiny nits. like with the dockerfile...we'll improve on this as we test/use it. thanks for picking up the work Evan.

**Comment by EckyYakov** [comment]
Made changes to License and README.md as well as replacing some references to the old repo

**Comment by EckyYakov** [comment]
Added in functionality for running the ntp test. updated documentation to reflect usage.

**Comment by EckyYakov** [comment]
Closing this so i can squash and make a clean commit

**Pull request closed without merging**